### DISCUSSION TITLE
Full screen that covers stage manager?

### DISCUSSION THREAD

**Discussion by kmthorsnes** [rxhanson/Rectangle#991] [General]
Hi, 
I see now that my `maximize`-shortcut, makes the screen not cover the stage manager. In a perfect world i would like it to also cover the stage manager (as it does with the dock) is this possible?

**Comment by ghost** [comment]
Go to the menu bar icon of Rectangle and open Settings. Then go to the last tab with the gear icon. 
At the bottom there is a slider (Stage Manager Area for last used Apps). Drag it all the way to the left (0 px).
